applications/crossbar/src/modules/cb_conferences.erl

```diff
@@ -216,6 +245,79 @@ load_conference(DocId, Context) ->
 %%--------------------------------------------------------------------
 %% @private
 %% @doc
+%% Load details of the participants of a conference
+%% @end
+%%--------------------------------------------------------------------
+-spec load_conference_details(cb_context:context(), path_token()) -> cb_context:context().
+load_conference_details(Context, ConfId) ->
+    AccountRealm = wh_util:get_account_realm(cb_context:account_db(Context), cb_context:account_id(Context)),
+    Req = [{<<"Realm">>, AccountRealm}
+           ,{<<"Fields">>, []}
+           ,{<<"Conference-ID">>, ConfId}
+           | wh_api:default_headers(?APP_NAME, ?APP_VERSION)
+          ],
+    ReqResp = whapps_util:amqp_pool_collect(Req
+                                            ,fun wapi_conference:publish_search_req/1
+                                            ,{'ecallmgr', 'true'}
+                                           ),
+    case ReqResp of
+        {'error', _} -> cb_context:add_system_error('conference_search_error', Context);
+        {_, JObjs} -> conference_participants(JObjs, Context)
+    end.
+
+-spec conference_participants(wh_json:objects(), cb_context:context()) -> cb_context:context().
+conference_participants([], Context) ->
+    cb_context:add_system_error('conference_not_active', Context);
+conference_participants([H|T]=ReqResp, Context) when is_list(ReqResp) ->
+    case wh_json:get_value(<<"Participants">>, H) of
+        'undefined' -> conference_participants(T, Context);
+        Participants -> participant_details(Participants, Context)
+    end;
+conference_participants(ReqResp, Context) ->
+    participant_details(wh_json:get_value(<<"Participants">>, ReqResp), Context).
+
+-spec participant_details(wh_json:objects(), cb_context:context()) -> cb_context:context().
+-spec participant_details(wh_json:objects(), cb_context:context(), wh_json:objects()) -> cb_context:context().
+participant_details(Participants, Context) ->
+    participant_details(Participants, Context, []).
+
+participant_details([], Context, Acc) ->
+    crossbar_doc:handle_json_success(Acc, Context);
+participant_details([H|T], Context, Acc) ->
+    CallId = wh_json:get_value(<<"Call-ID">>, H),
+    Req = [{<<"Call-ID">>, CallId}
+           | wh_api:default_headers(?APP_NAME, ?APP_VERSION)
+          ],
+    case whapps_util:amqp_pool_request(Req
```

Comment:
Just curious. Is there a way to group these requests into just one of size `length([H|T])`?